tests/test_ops/test_bilinear_grid_sample.py

```diff
@@ -24,17 +25,8 @@ def _test_bilinear_grid_sample(self,
         assert np.allclose(out.data.detach().cpu().numpy(),
                            ref_out.data.detach().cpu().numpy(), precision)
 
-    def test_bilinear_grid_sample(self):
-        self._test_bilinear_grid_sample(torch.double, False)
-        self._test_bilinear_grid_sample(torch.double, True)
-        self._test_bilinear_grid_sample(torch.float, False)
-        self._test_bilinear_grid_sample(torch.float, True)
-        self._test_bilinear_grid_sample(torch.float, False)
-        self._test_bilinear_grid_sample(torch.float, True, 5)
-        self._test_bilinear_grid_sample(torch.float, False, 10)
-        self._test_bilinear_grid_sample(torch.float, True, -6)
-        self._test_bilinear_grid_sample(torch.float, False, -10)
-        self._test_bilinear_grid_sample(torch.double, True, 5)
-        self._test_bilinear_grid_sample(torch.double, False, 10)
-        self._test_bilinear_grid_sample(torch.double, True, -6)
-        self._test_bilinear_grid_sample(torch.double, False, -10)
+    @pytest.mark.parametrize('dtype', [torch.double, torch.float])
+    @pytest.mark.parametrize('align_corners', [False, True])
+    @pytest.mark.parametrize('multiplier', [1, 10, -5])
+    def test_bilinear_grid_sample(self, dtype, align_corners, multiplier):
```

Comment:
It seems like that the unit tests are not consistent with the original.

Reply:
Yes, but I think the novel tests are also enough to test default, positive and negative numbers. We can add more `multiplier` options, but I think it is unnecessary.